mcc/NewGfx.c

```diff
@@ -76,12 +113,15 @@ LONG TextLengthNew(struct RastPort *rp, const char *string, ULONG count, LONG ta
 
 ///
 /// TextFitNew
-ULONG TextFitNew(struct RastPort *rp, const char *string, ULONG strLen, struct TextExtent *textExtent, const struct TextExtent *constrainingExtent, LONG strDirection, LONG constrainingBitWidth, LONG constrainingBitHeight, LONG tabSizePixels)
+ULONG TextFitNew(struct RastPort *rp, const char *string, ULONG strLen, struct TextExtentNew *textExtentNew, const struct TextExtent *constrainingExtent, LONG strDirection, LONG constrainingBitWidth, LONG constrainingBitHeight, LONG tabSizePixels)
 {
   ULONG result = 0;
   ULONG strLen0 = 0;
   const char *tptr = string;
   const char *tptr0 = string;
+  struct TextExtent textExtent;
+  ULONG te_width  = 0;
+  ULONG te_height = constrainingBitHeight < rp->TxHeight ? 0 : rp->TxHeight;
```

Comment:
move `te_height = constraining....` assignment after `ENTER()` and just leave variable definition here.